src/store/assets/fungible-tokens.ts

```diff
@@ -26,7 +26,7 @@ const fungibleTokenMetaDataState = atomFamily<
       });
       // can't return the promise directly
       return data;
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
Would `unknown` be a more accurate type here?

Reply:
I can try that, yep.

Reply:
This didn't work so will keep as `any` to get rid of the type error.